engine/engine.go

```diff
@@ -380,6 +380,8 @@ func (r *ruleEngine) createViolation(conditionResponse ConditionResponse, rule R
 		if r.incidentLimit != 0 && len(incidents) == r.incidentLimit {
 			break
 		}
+		// We need to determine if the incident is valid based on label selectors
```

Comment:
Is this a TODO or just a description of what's happening?

Reply:
errant TODO that is now down in the provider condition removing